src/main/java/org/sonarsource/sonarlint/ls/CommandManager.java

```diff
@@ -142,7 +142,7 @@ private static ResponseErrorException unknownRule(String ruleKey) {
   public void executeCommand(ExecuteCommandParams params, CancelChecker cancelToken) {
     switch (params.getCommand()) {
       case SONARLINT_UPDATE_ALL_BINDINGS_COMMAND:
-        bindingManager.updateAllBindings();
+        bindingManager.updateAllBindings(cancelToken, params.getWorkDoneToken());
```

Comment:
`params.getWorkDoneToken()` will always be null in VSCode because VSCode doesn't support client progress on commands